docs/user_guide/secure_payloads.rst

```diff
@@ -0,0 +1,93 @@
+Secure Payloads
+================
+
+.. warning::
+    This page is still under development and not complete. It will be so until
+    this warning is removed.
+
+Trusted payloads offers the ability to securely provision secrets to an enrolled node.
+
+The payload itself is sent to via the Keylime Tenant to the verifier and only when a node
+has passed its enrollment criteria (including any `tpm_policy` or IMA whitelist), will the 
+payload be passed to the agent for execution.
+
+k.  The scripts work by leveraging two features of keylime: automatic certificate
```

Comment:
It seems we are sometimes using "Keylime" sometimes "keylime" - any reason not to harmonize the writing ?